crates/re_types/src/datatypes/tensor_data_ext.rs

```diff
@@ -864,6 +875,11 @@ fn test_image_height_width_channels() {
         // h=1, w=1, RGB:
         (vec![1, 1, 3], Some([1, 1, 3])),
         (vec![1, 1, 1, 3], Some([1, 1, 3])),
+        (vec![1, 1, 3, 1], Some([1, 1, 3])),
+        //
+        // h=1, w=3, Mono:
+        (vec![1, 3, 1], Some([1, 3, 1])),
+        (vec![1, 3, 1, 1], Some([1, 3, 1])),
```

Comment:
This could just as well be `3, 1, 1` imho - I would not add it to the unit tests